SwiftTweaks/TweakTableCell.swift

```diff
@@ -72,11 +77,12 @@ internal final class TweakTableCell: UITableViewCell {
 	override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
 		super.init(style: .Value1, reuseIdentifier: reuseIdentifier)
 
-		[switchControl, stepperControl, colorChit, textField, disclosureArrow].forEach { accessory.addSubview($0) }
+		[switchControl, stepperControl, colorChit, textField, disclosureArrow, segmentedControl].forEach { accessory.addSubview($0) }
```

Comment:
Ultimately, I'd prefer that this selection happen in either a `UIPicker` or a pushed-on view controller, but a segmented control is a fine stopgap for now! Thanks for tackling this without any interaction designs upfront! 👍 👍 👍 👍